app/src/main/java/org/fossasia/openevent/general/auth/LoginActivityViewModel.kt

```diff
@@ -18,6 +18,10 @@ class LoginActivityViewModel(private val authService: AuthService) : ViewModel()
     fun isLoggedIn() = authService.isLoggedIn()
 
     fun login(email: String, password: String) {
+        if (email.isNullOrEmpty() || password.isNullOrEmpty()) {
+            progress.value = false
```

Comment:
Progress would be false already AFAIK

Send the error to view. "Email and Password cannot be empty"